modules/gateways/razorpay.php

```diff
@@ -39,6 +41,15 @@ function razorpay_config()
     global $CONFIG;
 
     $webhookUrl = $CONFIG['SystemURL'].'/modules/gateways/razorpay/razorpay-webhook.php';
+    $rzpOrderMapping = new RZPOrderMapping(razorpay_MetaData()["DisplayName"]);
```

Comment:
@rohitcbr , please use single quotes for each string

Reply:
used single quotes in all the places, https://github.com/razorpay/razorpay-whmcs/blob/5a11e2c69b505809069c3903bc58d89c216a5e85/modules/gateways/razorpay.php#L44